front/src/components/Gosling.tsx

```diff
@@ -44,6 +44,56 @@ export default class GoslingVis extends React.Component<Props, {}> {
           ...sample
         };
       });
+    
+    const outerPoints: [number, number][] = [];
+    const sorted = labelJSON.sort((a, b) => a.start - b.start);
+    sorted.forEach(({ start, end }, i) => {
+      const lb: [number, number] = [start, end]; // the left bottom point of a rectangle
+      if(outerPoints.length === 0 || end > outerPoints[outerPoints.length - 1][1]) {
+        // This means `lb` is an outer point
+        outerPoints.push(lb);
+      }
+    });
+    const overlayRects: Record<string, string | number>[] = [];
+    const template = { chromosome: 'chr5' }; // Support other chromosomes as well?
```

Comment:
Would it be needed to fade out entire chromosomes other than chromosome 5? I currently only showed the semi-transparent overlay to chromosome 5 as it looked like only chromosome 5 contained any data items.

Reply:
I only used chromosome 5 for the Hi-C dataset so only fading out chr5 should be fine.

Reply:
Do we add a data transform to only show the CHR 5 data?

Reply:
The Matrix is rendered directly by HiGlass which does not allow us to apply data transforms.

Maybe we can additionally overlay semi-transparent or opaque white rectangles on all chromosomes?

Reply:
Yes, you mentioned that before. Let's fade out other CHRs and this branch is ready to merge.
Thanks a lot for creating this PR!